UPGRADE.md

```diff
@@ -0,0 +1,23 @@
+# Upgrade Guide
+
+[2019-06-12] 1.1.x to 1.2.x
+---------------------------
+
+Config moved from `~/.twilio` (Unix) and `%USERPROFILE%\.twilio` (Windows) to `~/.twilio-cli` and `%USERPROFILE%\.twilio-cli`, respectively.
+
+To copy over your existing configuration, run the following commands (Unix):
+```
+rm -rf ~/.twilio-cli
+mv ~/.twilio/ ~/.twilio-cli/
+```
+
```

Comment:
For Windows:

rmdir /S %USERPROFILE%\.twilio-cli
move %USERPROFILE%\.twilio %USERPROFILE%\.twilio-cli

Reply:
Why not %APPDATA%\twilio-cli on Windows?